MyConfig.h

```diff
@@ -354,7 +354,7 @@
  * This feature is currently not supported for anything but RF24.
  * Require @ref MY_RF24_IRQ_PIN to be set.
  */
-//#define MY_RX_MESSAGE_BUFFER_FEATURE
+#define MY_RX_MESSAGE_BUFFER_FEATURE
```

Comment:
Won't defining this by default break a lot of existing installations (that don't have MY_RF24_IRQ_PIN defined, or don't have the irq pin connected)?

Reply:
Sure.

Reply:
Maybe define it by default for RFM but not for NRF? RFM will always have irq connected anyway, right? So only drawback is that the buffer uses more ram.

Reply:
Maybe for new rfm69 driver. New rfm69 driver is not often used in real setups I think.

Reply:
Agree with @mfalkvidd. This breaks NRF24 setups without IRQ pin connected and might influence timing for NRF24 setups.
Don't make it default in MyConfig.h.

Reply:
Also, comments say this only works for NRF24, which isn't true anymore after merging your PR ;-)